core/domain/user_services.py

```diff
@@ -551,3 +556,16 @@ def get_user_impact_score(user_id):
         return model.impact_score
     else:
         return 0
+
+def fetch_gravatar(email):
+    """Returns the gravatar corresponding to given email"""
```

Comment:
Yes, exactly. This would be a static, deterministic image that's used for all cases in which the fetch fails.